1906-05-25.xml

```diff
@@ -0,0 +1,6330 @@
+                        view to laying their grievances before H.H. the Khedive.</p>
+                </div>
+                <div type="item">
+                    <head>STEAMER MOVEMENTS.</head>
+                    <p>The SS. Trojan Prince, from Antwerp and London, with passengers and general
+                        cargo, left Malta yesterday, and is due at Alexandria on Monday.</p>
+                </div>
+                <cb n="6"/>
+                <div type="section">
+                    <head>NOTES FROM PORT SAID.</head>
+                    <div type="item">
+                        <head>THE TORPEDO DISASTER.</head>
+                        <p>(FROM OUR CORRESPONDENT).</p>
+                        <p>Port-Said, Thursday, </p>
+                        <p>The report last Friday morning that torpedo boat No. 56 had suffered a
+                            doleful disaster spread rapidly through the town. A few hours after the
+                            departure of the Astilla a storm of such severity arose as to delay the
+                            traffic in the Canal until midday. The news did not surprise those who
+                            are acquainted with the dangers of navigating such craft near this coast
+                            in rough weather.</p>
+                        <p>Torpedo boat No. 64, which, rumour had it, was the missing boat, did not
+                            proceed to sea with the others, having gone into dock to have her
+                            machinery overhauled. She is now in fighting trim again and will shortly
+                            rejoin her fellows at Malta.</p>
+                    </div>
+                    <div type="item">
+                        <head>Naval Drill.</head>
+                        <p>Detachments of the boys in Navy blue from H.M.S. Minerva occasionally go
+                            through drill on shore, a source of endless interest to natives and
+                            others. A familiar sight in our town now is their well known pet I bex,
+                            which is reported to have swallowed a London paper which has recently
+                            given great prominence to the prowess in swimming of some of those who
+                            went to El Arish. </p>
+                    </div>
+                    <div type="item">
+                        <head>German Troops.</head>
+                        <p>The North German Lloyd S.S. Neckar, hired transport, arrived on Friday
+                            from Tsing Tau with nearly 2,000 German troops. Contrary to the
+                            regulations in force in Egyptian ports the troops disembarked wearing
+                            side arms. However, on the protest of Schalk Bey, Commandant of the
+                            Police, to the German Consul the matter was rectified.</p>
+                    </div>
```

Comment:
this is _probably_ the end of the "Notes from Port Said" section...what do you think? The next item is not about the city of Port Said.